src/react-components/icons/svg/arrow-xsmall.svg

```diff
@@ -1 +1,3 @@
-<svg width="14" height="9" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.984 7.578L.424 1.514 1.9.163l5.22 5.695L12.341.163l1.475 1.351-5.56 6.064a1.542 1.542 0 01-2.272 0zm1.474-1.351z"/></svg>
\ No newline at end of file
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
FYI: The viewbox additions will block RN's release.  However since we don't release on Fridays, the earliest we'll need it is Monday.  I might end up extracting them to their own ticket if it becomes the last thing that's blocking, we can keep in touch about this on Slack